src/index.html

```diff
@@ -12,7 +12,49 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
```

Comment:
According to the checklist, you should add empty lines between multiline sibling blocks of HTML. It will add some 'air' and simplify reading. Please add empty lines between sibling `div` elements.